cmake/Info.plist.in

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should we name this file something more specific? for example, the default template file is called MacOSXFrameworkInfo.plist.in (https://cmake.org/cmake/help/latest/prop_tgt/MACOSX_FRAMEWORK_INFO_PLIST.html).
that conveys the additional information that this is for a Mac framework.

Reply:
Will do in my next PR